StepDefinitions/SkillFeatureStepDefinitions.cs

```diff
@@ -0,0 +1,107 @@
+using MarsOnboardingIC.Pages;
+using MarsOnboardingIC.Utilities;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using Reqnroll;
+using System;
+using Assert = NUnit.Framework.Assert;
+
+namespace MarsOnboardingIC.StepDefinitions
+{
+    [Binding]
+    public class SkillFeatureStepDefinitions
+    {
+        public IWebDriver driver;
+        public SkillFeatureStepDefinitions()
+        {
+            // Get driver instance from Hooks
+            driver = Hooks.driver;
+        }
+        [Given("I logged in as a register user")]
+        public void GivenILoggedInAsARegisterUser()
+        {
+
+            LoginPage loginPageObj = new LoginPage();
```

Comment:
you should move all page object creation into the constructor to make use of it please see  https://www.tutorialspoint.com/csharp/csharp_classes.htm

https://www.knowledgehut.com/tutorials/csharp/csharp-constructors